tools/qontract_cli.py

```diff
@@ -1070,8 +1082,22 @@ def cidr_blocks(ctx) -> None:
 
     cidrs.sort(key=lambda item: ipaddress.ip_network(item["cidr"]))
 
-    ctx.obj["options"]["sort"] = False
-    print_output(ctx.obj["options"], cidrs, columns)
+    if for_cluster:
+        latest_declared_addr = ipaddress.ip_address(
+            [item for item in cidrs if item["type"] == "cluster"][-1]["to"]
+        )
```

Comment:
this will throw error when there is no cluster cidr, better to find it explicitly then do None handling

```python
        latest_cluster_cidr = next(
            (item for item in reversed(cidrs) if item["type"] == "cluster"),
            None,
        )
```

Reply:
Updated and added None check